tests/playground.html

```diff
@@ -178,6 +179,65 @@
   }
 }
 
+function fakeDrag(id, dx, dy, opt_workspace) {
+  var ws = opt_workspace ? opt_workspace : Blockly.getMainWorkspace();
+  var blockToDrag = ws.getBlockById(id);
+
+  if (!blockToDrag) {
+    return;
```

Comment:
If a block is deleted during the fake drag routine, it exits, but leaves the queue intact.  Thus pushing the button again would resume the rountine, while adding another set of fake drags.  Pathological case: airstrike then fake drag interrupted by delete all blocks, then another airstrike.  Result, next 99 presses of fake drag do nothing, the 100th causes n^2 fake drags.
I think you either want to clear the queue before returning, or else schedule the next fake drag before returning.